internal/postgres.go

```diff
@@ -16,10 +16,22 @@
 
 package internal
 
-import "github.com/lib/pq"
+import (
+	"errors"
 
-// IsUniqueConstraintViolationErr returns true if the error is a postgresql unique_violation error
+	"github.com/lib/pq"
+	"github.com/mattn/go-sqlite3"
+)
+
+// IsUniqueConstraintViolationErr returns true if the error is a postgresql unique_violation or sqlite3.ErrConstraint
```

Comment:
We can't mix sqlite and postgres in the same file otherwise dendritejs will fail to compile (pq cannot be compiled to WASM). Keep the check contained to `sqlite` packages for now please.

Reply:
I thought since there's also a `postgres_wasm.go` (with `+build wasm`) that this file would be used when compiling for wasm.
Nevertheless, i've reverted the change and moved the check directly to the `sqlite` package.